src/ActionsTests/Actions/ScreenShotActionUnitTests.cs

```diff
@@ -29,127 +29,103 @@ private void SetBoundingRectangle(A11yElement element, Rectangle b)
-                    ElementGet = () => element,
-                    DataContextGet = () => dc,
+                    DataContext = dc,
                 };
 
-                ShimDataManager.GetDefaultInstance = () => new ShimDataManager
-                {
-                    GetElementContextGuid = (_) => elementContext,
-                };
+                dm.AddElementContext(elementContext);
 
-                ScreenShotAction.CaptureScreenShot(Guid.NewGuid());
+                ScreenShotAction.GetDataManager = () => dm;
+                
+                ScreenShotAction.CaptureScreenShot(elementContext.Id);
 
-                // screenshot is not set(null)
                 Assert.IsNull(dc.Screenshot);
-                // ScreenShotSet was not called.
-                Assert.IsFalse(bitmapsetcalled);
+                Assert.AreEqual(default(int), dc.ScreenshotElementId);
             }
         }
 
         [TestMethod]
         [Timeout(2000)]
         public void CaptureScreenShot_ElementWithBoundingRectangle_ScreenShotCreated()
         {
-            using (ShimsContext.Create())
+            using (var dm = new DataManager())
             {
-                bool bitmapsetcalled = false;
-
-                //  bounding rectangle exists.
                 A11yElement element = new A11yElement
                 {
-                    UniqueId = 1,
+                    UniqueId = 42,
                 };
-                SetBoundingRectangle(element, new Rectangle(0, 0, 10, 10));
 
-                ElementDataContext dc = new ShimElementDataContext()
-                {
-                    ScreenshotSetBitmap = (_) => bitmapsetcalled = true,
-                };
+                SetBoundingRectangle(element, new Rectangle(0, 0, 10, 10));
 
-                ElementContext elementContext = new ShimElementContext
-                {
-                    ElementGet = () => element,
-                    DataContextGet = () => dc,
-                };
+                var dc = new ElementDataContext(element, 1);
 
-                ShimDataManager.GetDefaultInstance = () => new ShimDataManager
+                var elementContext = new ElementContext(element)
                 {
-                    GetElementContextGuid = (_) => elementContext,
+                    DataContext = dc,
                 };
 
-                Graphics g = new ShimGraphics();
+                dm.AddElementContext(elementContext);
 
-                ShimGraphics.FromImageImage = (_) => g;
+                ScreenShotAction.GetDataManager = () => dm;
 
-                ScreenShotAction.CaptureScreenShot(Guid.NewGuid());
+                ScreenShotAction.CaptureScreenShot(elementContext.Id);
 
-                // bitmap is set
-                Assert.IsTrue(bitmapsetcalled);
+                Assert.IsNotNull(dc.Screenshot);
+                Assert.AreEqual(element.UniqueId, dc.ScreenshotElementId);
             }
         }
 
-        [TestMethod]
+            [TestMethod]
```

Comment:
An extra indent was added. Sugestion:
```suggestion
        [TestMethod]
```